docs/pipelineruns.md

```diff
@@ -158,17 +158,37 @@ spec:
 
 #### Tekton Bundles
 
-**Note: This is only allowed if `enable-tekton-oci-bundles` is set to
-`"true"` in the `feature-flags` configmap, see [`install.md`](./install.md#customizing-the-pipelines-controller-behavior)**
+A `Tekton Bundle` is an OCI artifact that contains Tekton resources like `Tasks` which can be referenced within a `taskRef`.
 
-You may also use a `Tekton Bundle` to reference a pipeline defined remotely.
+You can reference a `Tekton bundle` in a `TaskRef` in both `v1` and `v1beta1` using [remote resolution](./bundle-resolver.md#pipeline-resolution). The example syntax shown below for `v1` uses remote resolution, and requires enabling `enable-tekton-oci-bundles: "true"` feature flag.
```

Comment:
Is it true that using the bundles resolver requires this feature flag?

Reply:
That's what I interpreted from https://github.com/tektoncd/pipeline/blob/351d201b38197075bb1a0976de61679638457b7d/config/config-feature-flags.yaml#L69-L72

Reply:
Good catches. Let me ask on slack what's true.

Reply:
My guess is that `enable-tekton-oci-bundles` set to true is required in order to send the request to the bundle resolver while `enable-bundle-resolver` (which is true by default) is used to just enable it in the remote resolvers.

Reply:
It looks like `enable-tekton-oci-bundles` isn't needed for the bundle resolver: https://github.com/tektoncd/pipeline/blob/333b0d4f900b3896d0580e9d718a6497b1cc6626/pkg/reconciler/taskrun/resources/taskref.go#L117-L152

can you try it out?

Reply:
I ran the example: examples/taskruns/bundle_resolver.yaml by setting `enable-tekton-oci-bundles` to true and false and it works in both cases. So you're right Lee that it is actually not required. I wonder if I should make a followup PR removing that option from feature flags and update the corresponding docs then?

Reply:
Thanks for testing that out! I think this feature flag is still needed for bundle syntax, so we shouldn't remove it. You can just clarify in this PR that it's not needed for remote resolution.

Reply:
Done!